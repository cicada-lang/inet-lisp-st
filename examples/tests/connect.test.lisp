(define-node sole value!)

(define (main)
  (= first second (link))
  (connect first (sole))
  second)

(main)
