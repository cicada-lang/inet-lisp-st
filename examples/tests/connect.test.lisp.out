<net>
:root -<>-!value-(sole₁)
(sole₁
 :value! -<>-)
</net>

